java-lib/src/main/java/com/wavefront/common/MetricMangler.java

```diff
@@ -1,57 +1,58 @@
 package com.wavefront.common;
 
+import com.google.common.base.Splitter;
+
 import java.util.ArrayList;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+
 import javax.annotation.Nullable;
 
 /**
- * Handles updating the metric and source names by extracting components
- * from the metric name.
- * There are several options considered:
- *   - source name:
- *     - extracted from one or more components of the metric name
- *       (where each component is separated by a '.')
- *     - allow characters to be optionally replaced in the components extracted
- *       as source name with '.'
- *   - metric name:
- *     - remove components (in addition to the source name)
- *       (this allows things like 'hosts.sjc1234.cpu.loadavg.1m' to get
- *        changed to cpu.loadavg.1m after extracting sjc1234)
- *
+ * Handles updating the metric and source names by extracting components from the metric name. There
```

Comment:
Ah yes.  I reformatted using intellij.  Looks like it didn't like my non-html formatting.  I'll fix it so it's using html.